.eslintrc.js

```diff
@@ -56,19 +56,21 @@ module.exports = {
     "func-call-spacing": "error",
     "func-name-matching": "error",
     "func-names": "off",
-    "func-style": "error",
     "function-paren-newline": "error",
     "generator-star-spacing": "error",
-    "global-require": "error",
     "grouped-accessor-pairs": "error",
     "guard-for-in": "error",
-    "handle-callback-err": "error",
-    "id-blacklist": "error",
     "id-length": "error",
     "id-match": "error",
     "implicit-arrow-linebreak": "error",
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "indent-legacy": "off",
+    "indent": [
+      "error",
+      2,
+      {
+        "SwitchCase": 1,
+        "ignoredNodes": ["TemplateLiteral > *"]
```

Comment:
```suggestion
        "SwitchCase": 1,
        "ignoredNodes": ["TemplateLiteral > *"]
```

What is an example of the `ignoreNodes` rule that we can test this with? I am not sure if this syntax is correct.

Reply:
That looks right to me. To test, you can pull down my latest auro-interruption PR. Without these modifications, ESLint complains at the indentation in the `html` template literals.